apps/web/jest.config.ts

```diff
@@ -7,7 +7,7 @@ const config: Config.InitialOptions = {
   verbose: true,
   roots: ["<rootDir>"],
   setupFiles: ["<rootDir>/test/jest-setup.js"],
-  testMatch: ["**/test/lib/**/*.(spec|test).(ts|tsx|js)"],
+  testMatch: ["**/test/lib/**/*.(spec|test).(ts|tsx|js)", "**/__tests__/**/*.(spec|test).(ts|tsx|js)"],
```

Comment:
```suggestion
  testMatch: ["**/*.(spec|test).(ts|tsx|js)"],
```

Let's allow test to coexist with the rest of the code WDYT?

Reply:
I would go for "**/__tests__/**/*.(spec|test).(ts|tsx|js)" and then renaming test/lib/** to __test__/lib